python/python/lance/fragment.py

```diff
@@ -384,3 +385,76 @@ def metadata(self) -> FragmentMetadata:
         FragmentMetadata
         """
         return FragmentMetadata(self._fragment.metadata().json())
+
+
+def write_fragments(
+    data: ReaderLike,
+    dataset_uri: Union[str, Path],
+    schema: Optional[pa.Schema] = None,
+    *,
+    max_rows_per_file: int = 1024 * 1024,
+    max_rows_per_group: int = 1024,
+    max_bytes_per_file: int = 90 * 1024 * 1024 * 1024,
+    progress: Optional[FragmentWriteProgress] = None,
+) -> List[FragmentMetadata]:
+    """
```

Comment:
Why this function exposed via `Fragments` instead of dataset?

Reply:
Because it's a low-level API that involves manipulating individual fragments. It's the successor to `Fragment.create()`.